boswatch/inputSource/sdrInput.py

```diff
@@ -36,7 +36,7 @@ def _runThread(self, dataQueue, sdrConfig, decoderConfig):
             sdrProc.addArgument("-p " + str(sdrConfig.get("error", default="0")))      # frequency error in ppm
             sdrProc.addArgument("-l " + str(sdrConfig.get("squelch", default="1")))    # squelch
             sdrProc.addArgument("-g " + str(sdrConfig.get("gain", default="100")))     # gain
-            if sdrConfig.get("fir_size", default=None):
+            if (sdrConfig.get("fir_size") is not None):
```

Comment:
Wieso `default` entfernt?

Reply:
Bei dem Versuch in der client.yaml den Wert für die fir_size auf 0 zu setzen, wird bei rtl_fm kein Parameter ``-F 0`` übergeben. Nur bei der Bedingung ``is not None`` funktioniert das zuverlässig.

Scheinbar wird fir_size: 0 als false interpretiert und es wird kein Wert ausgegeben. Dieser Fix greift bei explizit beim Wert 0 ein. Bitte probiere es aus.

Reply:
Nein, was ich meinte ist:
Wieso wurde der `default` Tag entfernt?
Wenn du die option `fir_size` aus der config entfernst,
sollte ja der default Wert `None` sauber geliefert werden.
Dafür ist diese default Section im `config.get()` da.

Reply:
Hier ist die Doku falsch. Das default gehört wenn dann in die hintere Sektion.
https://github.com/BOSWatch/BW3-Core/blob/develop/docu/docs/config.md?plain=1#L52
Werte mit Default können in der config auch einfach weggelassen werden.
Damit das funktioniert muss beim `config.get()` ein entsprechender Default Wert auch angegeben sein.
Deine `if`-Clause muss dann eben so gestaltet sein, dass dies sauber funktioniert.

Reply:
Der Wert ist jetzt korrigiert und die Doku entsprechend angepasst. Ich hoffe es ist jetzt so wie es soll.

Reply:
Ja, so meinte ich das - Danke fürs anpassen.
Ich gehe davon aus, so funktioniert das noch wie gewollt?
Wenn ich's gar nicht angebe wird's auch nicht übergeben. 0 sollte gehen und jede andere Zahl natürlich auch.

Reply:
Ja, ich habe es gestern erfolgreich getestet.
Vielen Dank für deine Unterstützung.